devel/turbogit/Portfile

```diff
@@ -13,17 +13,18 @@ maintainers         {b4nst @b4nst} openmaintainer
 description         Your git boat friend
 long_description    Cli tool built to help you deal with your day-to-day git work.
 
-github.tarball_from releases
-worksrcdir          .
+github.tarball_from archive
 
-checksums           rmd160  3f074b41b44ecad49f0e267bbb1b1d9ba2139cef \
-                    sha256  061a2caa6bf4ec658c9b584167a6ecf75eeb1f51b07a23377953b4d7ee27de8a \
-                    size    1138091
+checksums           rmd160  9f2d13d0212591531cfe8a296023b2efe8187c64 \
+                    sha256  8f0facbd3cdc331efea8a11871e1556b572a1d4b8f89761836252c0b0f4b2d05 \
+                    size    1168134
 
 use_configure       no
 installs_libs       no
 
-depends_build       port:go
+depends_build       port:go \
+                    port:pkgconfig \
+                    port:libgit2
```

Comment:
it appears to me that `libgit2` is also required at runtime (and not only for building it):
```
% otool -L /opt/local/bin/tug
/opt/local/bin/tug:
	/opt/local/lib/libgit2.1.0.dylib (compatibility version 1.0.0, current version 1.0.1)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1281.100.1)
	/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (compatibility version 150.0.0, current version 1677.104.0)
	/System/Library/Frameworks/Security.framework/Versions/A/Security (compatibility version 1.0.0, current version 59306.140.5)

% port provides /opt/local/lib/libgit2.1.0.dylib
/opt/local/lib/libgit2.1.0.dylib is provided by: libgit2
```

so you should have that one listed as `depends_lib` instead.

Reply:
Wow I'm tired! Thanks and yes obviously it's required at runtime, I'm using go binding... My bad this PR was a mess.